privacyidea/lib/token.py

```diff
@@ -2041,7 +2042,8 @@ def check_token_list(tokenobject_list, passw, user=None, options=None):
                 # Token is locked
                 pin_match = False
                 otp_count = -1
-                repl = {'message': tae.message}
+                _num, message = tae.args
+                repl = {'message': message}
```

Comment:
Hmm, Google told me ``e.message`` is deprecated for ``Exception``, but wouldn't we be fine in case of ``TokenAdminError`` because we add the ``message`` attribute ourselves?

Reply:
Ok, actually i didn't check that :-) just the message from pylint...

Reply:
:+1: